cypress/support/step-definitions/show-edit-pod-steps.js

```diff
@@ -60,19 +60,27 @@ Then('Show Edit Pod component on a secondary block has no border property', () =
     .should('have.css', 'border', '0px none rgba(0, 0, 0, 0.85)');
 });
 
-Then('Show Edit Pod component has a cancel button', () => {
+Then('Show Edit Pod classic component has a cancel button', () => {
   showEditPodCancelButton()
     .should('be.visible')
     .and('have.css', 'background', 'rgba(0, 0, 0, 0) none repeat scroll 0% 0% / auto padding-box border-box')
     .and('have.css', 'border-color', 'rgb(37, 91, 199)')
     .and('have.css', 'color', 'rgb(37, 91, 199)');
 });
 
+Then('Show Edit Pod component has a cancel button', () => {
+  showEditPodCancelButton()
+    .should('be.visible')
+    .and('have.css', 'background', 'rgba(0, 0, 0, 0) none repeat scroll 0% 0% / auto padding-box border-box')
+    .and('have.css', 'border-color', 'rgb(0, 128, 93)')
+    .and('have.css', 'color', 'rgb(0, 128, 93)');
```

Comment:
and use it here 
```suggestion
    .and('have.css', 'color', color);
```
in this case there is no need to create a seprate step for classic story

Reply:
fixed